guide/web-scene.md

```diff
@@ -0,0 +1,75 @@
+## X3D and Web Scene
+
+### Description
+
+Webots can export a world to an interactive 3D `HTML` page.
+This feature is especially useful for publishing Webots-created worlds on the Web.
+
+During the export, a [X3D](http://www.web3d.org/x3d/what-x3d) file and a `HTML5` page are generated.
+
+The `X3D` file contains the graphical information of the world.
+`X3D` is a modern `XML`-based file format for representing 3D computer graphics.
+It can be imported in many 3D modeling or visualizing applications.
+
+The `HTML5` page contains a Webots player which displays the `X3D` file
+as shown in the [figure below](#screenshot-of-a-web-scene-page-generated-by-webots).
+It can be open in any recent Web browser as documented in the [section below](#remarks-on-the-used-technologies-and-their-limitations).
+The navigation in the player is done using the mouse or the touch screen, similarly to the Webots navigation.
+
+%figure "Screenshot of a Web scene page generated by Webots"
+
+![screenshot-web-scene.png](images/screenshot-web-scene.png)
+
+%end
+
+
+### How to export a Web Scene
+
+Select the `File / Export HTML5 Model...` menu item and choose the target `HTML` file.
+Webots will ask to playback the resulting file in the default browser.
+
+**Note**:
+The `X3D` file and the required textures are exported in the same directory as the target `HTML` file.
+
+**Note**:
+The playback option may not work correctly depending on your default Web browser.
+In this case, please refer to the [section below](#remarks-on-the-used-technologies-and-their-limitations).
+
+
+### How to embed a Web Scene in your Website
+
+The exported `HTML` page is designed to be as simple as possible, and is the reference for
+an integration in an external Website.
+Alternatively, an `<iframe>` tag pointing to the generated Webots page is a less elegant but simpler solution.
+
+The resources (`CSS`, `JavaScript`, etc.) on the [Cyberbotics Website](https://www.cyberbotics.com)
+will be stored for long term, and can be used from an external Website.
+
+
+### Limitations
+
+All the graphical nodes (Box, Sphere, Appearance, Material, ImageTexture, Light, Transform, etc.) are supported.
+
+The rendering of the Webots player is done using the [X3DOM](http://www.x3dom.org) Web standard.
+It may occur that the rendering is not strictly equivalent, because our both implementations of the `X3D` standard may differ.
```

Comment:
II would explicitly specify which rendering are not equivalent.
Something like
``It may occur that the rendering in the Webots application and in the exported Web page are not strictly equivalent``

and I would personally drop the 'our both'